tests/consumers/acero_consumer.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+from pathlib import Path
+
+import pyarrow as pa
+
+try:
+    import pyarrow.substrait as substrait
+except ImportError:
+    substrait = None
+
+
+class AceroConsumer:
+    """
+    Implementation of the acero substrait consumer Class for testing
+    """
+    def __init__(self):
+        pass
```

Comment:
why just pass?

Reply:
I actually don't even need this.  I'll remove it.